src/fsharp/FSharp.Core/list.fs

```diff
@@ -100,7 +100,7 @@ namespace Microsoft.FSharp.Collections
                 loop ([], state) (rev list)
 
         [<CompiledName("Iterate")>]
-        let iter action list = Microsoft.FSharp.Primitives.Basics.List.iter action list
+        let inline iter action (list:'T list) = for x in list do action x
```

Comment:
Nice. A bit surprised that these older, unoptimized functions for lists are still in use.